test/utilities/connectivity_testing.go

```diff
@@ -237,15 +239,20 @@ func CreateGatewayEp(networkId string, ipAddress string, ipv6Adress string) erro
 	if err != nil {
 		return fmt.Errorf("Gateway Endpoint Not Found: %v", err)
 	}
-	ep.HostAttach(1)
+	if err := ep.HostAttach(1); err != nil {
+		return fmt.Errorf("Failed to attach HNS endpoint %q to host: %s", createdEp.Name, err)
+	}
 
 	//Hard Code for now
 	vNicName := fmt.Sprintf(`"vEthernet (%s)"`, gwEp.Name)
 	vEthernet := `"vEthernet (Ethernet)"`
 	os.Setenv("vEthernet", vEthernet)
 	os.Setenv("vNicName", vNicName)
-	cmd := exec.Command("cmd", "/c", "netsh", "int", "ipv4", "set", "int", "%vNicName%", "for=en")
-	cmd.Run()
+	command := []string{"cmd", "/c", "netsh", "int", "ipv4", "set", "int", "%vNicName%", "for=en"}
+	cmd := exec.Command(command[0], command[1:]...)
```

Comment:
If you are splitting here, is it really worth to make a command list in the previous line

Reply:
Indeed it was an overly-complicated way to store the whole command string so I could report in in an error below.

I've updated the code to simply re-format the command in the error using the `exec.Cmd.{Path,Args}` fields.